### PR TITLE
Fix company + Live Search User

### PR DESCRIPTION
**Pull request by vuminhhieu1311**
## Related Tickets
- [#40170](https://edu-redmine.sun-asterisk.vn/issues/40170)

## WHAT (optional)
- Live Search User Accounts
- Fix Company Job List Page
## Evidence (Screenshot or Video)
ezgif-2-7180367dccfa
Screen Shot 2021-08-15 at 10 32 03

### CODE REVIEW

**Review by nvvu99 — APPROVED** [review]
Ok

**Comment by vuminhhieu1311** [comment]
ready

File: app/Http/Controllers/Auth/LoginController.php [hunk 1]
```diff
@@ -33,9 +33,9 @@ protected function redirectTo()
 
         if ($isActivated == config('user.status.active')) {
             if ($role == config('user.admin')) {
-                //return admin dashboard
+                return route('admin.dashboard');
             } elseif ($role == config('user.employee')) {
-                return route('jobs.index');
+                return route('home');
             } elseif ($role == config('user.employer')) {
                 return route('home');
```
Comment (huyenttm-0901) [inline app/Http/Controllers/Auth/LoginController.php:40]: 2 case này cùng return 1 view à? gộp vào em

File: resources/lang/en/messages.php [hunk 2]
```diff
@@ -180,9 +180,16 @@
...
     'edit-comment' => 'Edit comment',
     'current-password' => 'Current password',
     'update' => 'Update',
     'update-email-password' => 'Update your email address and password',
+=======
+    'close' => 'Close',
+    'open' => 'Open',
+    'created-at' => 'Created At',
+    'searching-for' => 'You are Searching for',
+>>>>>>> b9d1656 (Live Search Users + Fix company jobs list page)
```
Comment (huyenttm-0901) [inline resources/lang/en/messages.php:194]: này em

File: resources/lang/vi/messages.php [hunk 3]
```diff
@@ -181,9 +181,16 @@
...
     'edit-comment' => 'Sửa bình luận',
     'current-password' => 'Mật khẩu hiện tại',
     'update' => 'Cập nhật',
     'update-email-password' => 'Cập nhật địa chỉ email và mật khẩu',
+=======
+    'close' => 'Đóng',
+    'open' => 'Mở',
+    'created-at' => 'Ngày Tạo',
+    'searching-for' => 'Bạn đang tìm kiếm',
+>>>>>>> b9d1656 (Live Search Users + Fix company jobs list page)
```
Comment (huyenttm-0901) [inline resources/lang/vi/messages.php:195]: tương tự

**Comment by vuminhhieu1311** [comment]
ready

**Review by huyenttm-0901 — APPROVED** [review]

**Pull request merged**